spec.html

```diff
@@ -37836,6 +37836,63 @@ <h1>RegExp.prototype</h1>
         <p>This property has the attributes { [[Writable]]: *false*, [[Enumerable]]: *false*, [[Configurable]]: *false* }.</p>
       </emu-clause>
 
+      <emu-clause id="sec-regexp.escape" number="2">
+        <h1>RegExp.escape ( _S_ )</h1>
```

Comment:
The `RegExp.escape` section should come before `RegExp.prototype` (alphabetical order).